src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -37280,8 +37280,6 @@ public class Derived : Base
 }
 ";
             var comp = CreateNullableCompilation(new[] { NotNullWhenAttributeDefinition, source });
-            // Note: we're missing warnings on F1 because the overridden method with substituted T from Derived has an oblivious T
-            // https://github.com/dotnet/roslyn/issues/41368
```

Comment:
It doesn't look like new diagnostics were added here. Was this issue already solved before this PR?

Reply:
The comment was incorrect or out-of-date (we already had the appropriate warnings on F1 for this scenario)

---
In reply to: [536444553](https://github.com/dotnet/roslyn/pull/49723#discussion_r536444553) [](ancestors = 536444553)